dora/core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -302,6 +302,14 @@ default List<PageId> getCachedPageIdsByFileId(String fileId, long fileLength) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Invalidates all pages by its file id.
+   * @param fileId the file id of the target file
+   */
+  default void invalidateAllPagesByFileId(String fileId) {
+    // Do nothing;
+  }
```

Comment:
1. Rename to `deleteFile` to be consistent with existing `deletePage` method, as `invalidation` is not something `CacheManager` understands. Also there is an existing `invalidate` method, which can be confused with this method.
2. Change to required method. 

```suggestion
  /**
   * Deletes all pages of the given file.
   * @param fileId the file id of the target file
   */
  void deleteFile(String fileId);
```

Reply:
OK.